DateTimeMate.go

```diff
@@ -92,3 +118,29 @@ func Reformat(source string, outputFormat string) (string, error) {
 	}
 	return f.FormatString(s), nil
 }
+
+// UnixStringToTime converts a string containing a Unix timestamp to time.Time.
+// It accepts timestamps in both seconds (10 digits) and milliseconds (13 digits).
+// Returns the corresponding time.Time and any error encountered during conversion.
+//
+// If the input string is not a valid integer or is empty,
+// it returns a zero time.Time and an error.
+func UnixStringToTime(timestamp string) (time.Time, error) {
+	unixTime, err := strconv.ParseInt(timestamp, 10, 64)
+	if err != nil {
+		return time.Time{}, err
+	}
+
+	if len(timestamp) == 13 {
+		return time.UnixMilli(unixTime), nil
+	}
+
+	return time.Unix(unixTime, 0), nil
+}
+
+// isPureIntegerAtoi reports whether a string contains a valid base-10 integer.
+// It returns true only if the string can be fully converted to an integer.
+func isPureIntegerAtoi(s string) bool {
+	_, err := strconv.Atoi(s)
```

Comment:
I made the following adjustment:

```go
func Reformat(source string, outputFormat string) (string, error) {
	source = strings.TrimSpace(source)
	if isPureIntegerAtoi(source) {
		if source[0] == '-' {
			return "", fmt.Errorf("timestamps can't be negative: %v", source)
		}
		t, err := unixStringToTime(source)
		if err != nil {
			return "", err
		}
		source = t.String()
	} else {
		source = ConvertRelativeDateToActual(source)
	}
```

```shell
# notice the --
$ go run cmd/dtmate/main.go fmt -- -1731877834 "%F %T" 
timestamps can't be negative: -1731877834
exit status 1

# without the --
$ go run cmd/dtmate/main.go fmt -1731877834 "%F %T"
Error: unknown shorthand flag: '1' in -1731877834
```

I can add the following line to `fmt.go`:

```go
DisableFlagParsing: true, // this allows for negative durations
```

Similar to what I did in [conv.go](https://github.com/jftuga/DateTimeMate/blob/main/cmd/dtmate/cmd/conv.go#L17)

The problem is that if I add this line of code then this command, this problem occurs:

```shell
$  go run cmd/dtmate/main.go fmt -l
Error: requires two arguments: [date/time] [format specifiers]
```

Ideally, I could cleanly check for a negative value when a user does not use `--` as well as let the `-l` option work properly: emit a list of format modifiers.

What do you think the best approach to resolving this might be?